File: multitextwatcher/src/main/java/com/mpt/android/multitextwatcher/OnTextWatcher.java
package com.mpt.android.multitextwatcher;

import android.text.Editable;
import android.widget.EditText;

public interface OnTextWatcher {
    void beforeTextChanged(EditText editText, CharSequence s, int start, int count, int after);

    void onTextChanged(EditText editText, CharSequence s, int start, int before, int count);

    void afterTextChanged(EditText editText, Editable editable);
}

File: multitextwatcher/src/main/java/com/mpt/android/multitextwatcher/OnTextWatcher.kt
package com.mpt.android.multitextwatcher

import android.text.Editable
import android.widget.EditText

interface OnTextWatcher {
    fun beforeTextChanged(editText: EditText, s: CharSequence, start: Int, count: Int, after: Int)
    fun onTextChanged(editText: EditText, s: CharSequence, start: Int, before: Int, count: Int)
    fun afterTextChanged(editText: EditText, editable: Editable)
}